src/loadr.cpp

```diff
@@ -0,0 +1,83 @@
+/* ****************************************************************************
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+*
+*
+* This file is part of Microsoft R Host.
+*
+* Microsoft R Host is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Microsoft R Host is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Microsoft R Host.  If not, see <http://www.gnu.org/licenses/>.
+*
+* ***************************************************************************/
+
+#define RHOST_NO_API_REDIRECT
+#include "stdafx.h"
+
+HMODULE r_module = nullptr;
+HMODULE rgraphapp_module = nullptr;
+
+#define _GET_PROC(m, api) _RAPI_PTR(api) = get_proc<decltype(api)>(m, _RAPI_STR(api));
```

Comment:
Identifiers starting with underscore followed by capital letter are reserved to the compiler and the standard library.

In this case, since these macros are contained within the .cpp file rather than a header, there's no reason to obfuscate their names.